File: OPENAPI-REST-API/swagger-client/scala-akka-http-server/src/main/scala/io/swagger/server/model/SuccessTextResponse.scala
package io.swagger.server.model


/**
 * = SuccessTextResponse =
 *
 * Response with success flag and text description.
 *
 * @param success Indicates whether or not the command was successful or not.
 * @param text Text associated with the response.
 * @param action Optional Action relating to the response.
 */
case class SuccessTextResponse (
  success: Boolean,
  text: Option[String],
  action: Option[String]
)

File: OPENAPI-REST-API/openapi-client/php-dt/src/App/DTO/SuccessTextResponse.php
<?php
declare(strict_types=1);

namespace App\DTO;

use Articus\DataTransfer\PhpAttribute as DTA;

/**
 * Response with success flag and text description.
 */
class SuccessTextResponse
{
    /**
     * Indicates whether or not the command was successful or not.
     */
    #[DTA\Data(field: "success")]
    #[DTA\Validator("Scalar", ["type" => "bool"])]
    public bool|null $success = null;

    /**
     * Text associated with the response.
     */
    #[DTA\Data(field: "text", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $text = null;

    /**
     * Optional Action relating to the response.
     */
    #[DTA\Data(field: "action", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $action = null;

}
